From e7ed0c3eaa26fde4132f82363f9d587e4256d718 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Tue, 18 Aug 2015 15:53:54 +0800
Subject: [PATCH 0236/5242] MLK-11375-2 ARM: imx: disable cpuidle for imx6sx

commit  0e30f0dfc2df4d1143e988d6fa107b4825588740 from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX6SX CPUIdle with ARM core power down has issue
and cause system hang during kernel boot up, disable
it for now and will enable it when busfreq driver
is ready.

Signed-off-by: Anson Huang <b20788@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/mach-imx6sx.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/mach-imx/mach-imx6sx.c b/arch/arm/mach-imx/mach-imx6sx.c
index 3f8b0df..42a2639 100644
--- a/arch/arm/mach-imx/mach-imx6sx.c
+++ b/arch/arm/mach-imx/mach-imx6sx.c
@@ -100,8 +100,6 @@ static void __init imx6sx_init_irq(void)
 
 static void __init imx6sx_init_late(void)
 {
-	imx6sx_cpuidle_init();
-
 	if (IS_ENABLED(CONFIG_ARM_IMX6Q_CPUFREQ))
 		platform_device_register_simple("imx6q-cpufreq", -1, NULL, 0);
 }
-- 
1.7.9.5

